internal/service/common/device.go

```diff
@@ -92,6 +92,13 @@ func UpdateServiceSideStatus(ctx context.Context, st store.Store, log logrus.Fie
 
 func updateServerSideDeviceStatus(device *api.Device) bool {
 	lastDeviceStatus := device.Status.Summary.Status
+	if device.IsDecommissioning() {
+		device.Status.Lifecycle = api.DeviceLifecycleStatus{
+			Info:   util.StrToPtr("Device has acknowledged decommissioning request"),
+			Status: api.DeviceLifecycleStatusDecommissioning,
+		}
+		// todo - not sure if we can just return here or we need to go through other updates
```

Comment:
as per conv we should report/update the status until  we deem the device "dead,gone, etc"